### PR TITLE
Rebranding

### PR DESCRIPTION
**Pull request by fboucquez**
Rebranding for issue https://github.com/nemtech/symbol-openapi-generator/issues/22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
-The official NEM Api descriptor can be found [here](https://github.com/nemtech/nem2-docs/blob/master/source/resources/collections/openapi3.yaml):
+Project in charge of generating the API client libraries used by the NEM2-SDKs.
 
-# Requirements
+The ``generate`` script creates different flavours of clients according to the [current OpenAPI specification](https://github.com/nemtech/nem2-openapi).
+
+## Supported languages
+| Language   | Flavours          | 
+|------------|------------------|
+| Javascript | typescript-node  |
```
Comment (rg911) [inline README.md]: Language: ->Typescript

File: README.md [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
+[![Build Status](https://travis-ci.org/nemtech/symbol-openapi-generator.svg?branch=master)](https://travis-ci.org/nemtech/symbol-openapi-generator)
 
-The official NEM Api descriptor can be found [here](https://github.com/nemtech/nem2-docs/blob/master/source/resources/collections/openapi3.yaml):
+Project in charge of generating the API client libraries used by the NEM2-SDKs.
 
-# Requirements
+The ``generate`` script creates different flavours of clients according to the [current OpenAPI specification](https://github.com/nemtech/nem2-openapi).
+
+## Supported languages
+| Language   | Flavours          |
```
Comment (rg911) [inline README.md]: I would use `Template` instead of `Flavours` ?

File: README.md [hunk 3]
```diff
@@ -1,18 +1,29 @@
...
-* Java 8 - One way of installing is using [sdkman](https://sdkman.io/)
+* Java 8
 
-# How to use it
+## Usage
 
-Once tools have been installed, just run
+Once you have installed the dependencies, run:
 
 ~~~~
```
Comment (rg911) [inline README.md:27]: Remove

File: README.md [hunk 4]
```diff
@@ -1,18 +1,29 @@
...
 
 * NPM 8
-* Java 8 - One way of installing is using [sdkman](https://sdkman.io/)
+* Java 8
 
-# How to use it
+## Usage
 
-Once tools have been installed, just run
+Once you have installed the dependencies, run:
```
Comment (rg911) [inline README.md:26]: To install the dependencies, run:

File: README.md [hunk 5]
```diff
@@ -21,20 +32,19 @@ npm install @openapitools/openapi-generator-cli@cli-4.1.0 -g
 
 The script will:
 
-1. Generate one lib per flavor/framework/library. Currently typescript-node, vertx, jersey2 and okhttp-gson (see possible options for java [here](https://openapi-generator.tech/docs/generators/java) and for javascript [here](https://openapi-generator.tech/docs/generators/javascript))
-2. Build each library
-3. Build and install libraries using NPM and Gradle
+1. Generate one library per flavor/framework/library.
```
Comment (rg911) [inline README.md]: Generate one library per language / template

**Comment by fboucquez** [comment]
Travis has updated itself automatically

https://travis-ci.org/nemtech/symbol-openapi-generator

But unsure yet if the encrypted keys used to deploy to maven will work once merged to master

**Review by rg911 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (fboucquez) [reply README.md]: ok

File: README.md (hunk 1 above)
Comment (fboucquez) [reply README.md]: ok

File: README.md (hunk 3 above)
Comment (fboucquez) [reply README.md:27]: The line?

File: README.md (hunk 4 above)
Comment (fboucquez) [reply README.md:26]: the dependencies are NPM and Java listed above

**Comment by fboucquez** [comment]
Artifact renaming:

| Current Name | New Name |
| ------------- | ------------- |
| io.nem:api-jersey2-client  | io.nem:symbol-openapi-jersey2-client |                                 
| io.nem:api-okhttp-gson-client  | io.nem:symbol-openapi-okhttp-gson-client |                                 
| io.nem:api-vertx-client   | io.nem:symbol-openapi-vertx-client  |                                    
| api-typescript-node-client   | symbol-openapi-typescript-node-client  |

File: README.md (hunk 5 above)
Comment (fboucquez) [reply README.md]: ok

File: README.md (hunk 3 above)
Comment (rg911) [reply README.md:27]: empty line

**Comment by fboucquez** [comment]
This PR depends on the [nem2-open api](https://github.com/nemtech/nem2-openapi) rebranding. Once the rebranding is done, we can update the open api spec here. @dgarcia360 issue https://github.com/nemtech/nem2-openapi/issues/82

**Review by rg911 — APPROVED** [review]

**Pull request merged**